xtext.web + npm + windows

```
Failed to execute mojo com.github.eirslett:frontend-maven-plugin:1.12.1:npm {execution: optimize} (com.github.eirslett:frontend-maven-plugin:1.12.1:npm:optimize:generate-resources)

org.eclipse.core.runtime.CoreException: Failed to execute mojo com.github.eirslett:frontend-maven-plugin:1.12.1:npm {execution: optimize}
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeMojo(MavenExecutionContext.java:340)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.lambda$0(MavenExecutionContext.java:291)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeBare(MavenExecutionContext.java:394)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:275)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:290)
	at org.eclipse.m2e.core.project.configurator.MojoExecutionBuildParticipant.build(MojoExecutionBuildParticipant.java:57)
	at org.eclipse.m2e.core.internal.builder.MavenBuilderImpl.lambda$1(MavenBuilderImpl.java:139)
	at java.base/java.util.LinkedHashMap.forEach(LinkedHashMap.java:721)
	at org.eclipse.m2e.core.internal.builder.MavenBuilderImpl.build(MavenBuilderImpl.java:122)
	at org.eclipse.m2e.core.internal.builder.MavenBuilder$1.method(MavenBuilder.java:164)
	at org.eclipse.m2e.core.internal.builder.MavenBuilder$1.method(MavenBuilder.java:1)
	at org.eclipse.m2e.core.internal.builder.MavenBuilder$BuildMethod.lambda$1(MavenBuilder.java:109)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeBare(MavenExecutionContext.java:394)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:228)
	at org.eclipse.m2e.core.internal.builder.MavenBuilder$BuildMethod.lambda$0(MavenBuilder.java:100)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeBare(MavenExecutionContext.java:394)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:275)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:214)
	at org.eclipse.m2e.core.internal.builder.MavenBuilder$BuildMethod.execute(MavenBuilder.java:83)
	at org.eclipse.m2e.core.internal.builder.MavenBuilder.build(MavenBuilder.java:192)
	at org.eclipse.core.internal.events.BuildManager$2.run(BuildManager.java:1020)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:247)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:303)
	at org.eclipse.core.internal.events.BuildManager$1.run(BuildManager.java:392)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:395)
	at org.eclipse.core.internal.events.BuildManager.basicBuildLoop(BuildManager.java:506)
	at org.eclipse.core.internal.events.BuildManager.basicBuildLoop(BuildManager.java:454)
	at org.eclipse.core.internal.events.BuildManager.build(BuildManager.java:536)
	at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:196)
	at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:289)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: org.apache.maven.plugin.MojoFailureException: Failed to run task
	at com.github.eirslett.maven.plugins.frontend.mojo.AbstractFrontendMojo.execute(AbstractFrontendMojo.java:100)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeMojo(MavenExecutionContext.java:338)
	... 32 more
Caused by: com.github.eirslett.maven.plugins.frontend.lib.TaskRunnerException: 'npm run optimize' failed.
	at com.github.eirslett.maven.plugins.frontend.lib.NodeTaskExecutor.execute(NodeTaskExecutor.java:63)
	at com.github.eirslett.maven.plugins.frontend.mojo.NpmMojo.execute(NpmMojo.java:62)
	at com.github.eirslett.maven.plugins.frontend.mojo.AbstractFrontendMojo.execute(AbstractFrontendMojo.java:94)
	... 34 more
Caused by: com.github.eirslett.maven.plugins.frontend.lib.ProcessExecutionException: org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
	at com.github.eirslett.maven.plugins.frontend.lib.ProcessExecutor.execute(ProcessExecutor.java:82)
	at com.github.eirslett.maven.plugins.frontend.lib.ProcessExecutor.executeAndRedirectOutput(ProcessExecutor.java:64)
	at com.github.eirslett.maven.plugins.frontend.lib.NodeExecutor.executeAndRedirectOutput(NodeExecutor.java:29)
	at com.github.eirslett.maven.plugins.frontend.lib.NodeTaskExecutor.execute(NodeTaskExecutor.java:58)
	... 36 more
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
	at com.github.eirslett.maven.plugins.frontend.lib.ProcessExecutor.execute(ProcessExecutor.java:74)
	... 39 more
```

either the downloaded npm is not called or needs external install

--- a/org.eclipse.xtext.web/package.json
+++ b/org.eclipse.xtext.web/package.json
@@ -9,16 +9,16 @@
 		"test": "mocha -u tdd --reporter dot src/test/js/*.js",
 		"optimize": "npm run optimizeAce && npm run optimizeOrion && npm run optimizeCodeMirror",
 		"optimizeAce": "npm run optimizeAceJs && npm run optimizeAceJsMin && npm run optimizeAceCss",
-		"optimizeAceJs":"r.js -o src/main/js/requirejs-ace-config.js logLevel=2 out=build/opt-resources/xtext-ace.js optimize=none",
-		"optimizeAceJsMin": "r.js -o src/main/js/requirejs-ace-config.js logLevel=2 out=build/opt-resources/xtext-ace.min.js optimize=uglify preserveLicenseComments=false",
-		"optimizeAceCss": "r.js -o logLevel=2 cssIn=src/main/css/xtext/xtext-ace.css out=build/opt-resources/xtext-ace.css",
+		"optimizeAceJs":"node node_modules/requirejs/bin/r.js -o src/main/js/requirejs-ace-config.js logLevel=2 out=build/opt-resources/xtext-ace.js optimize=none",
+		"optimizeAceJsMin": "node node_modules/requirejs/bin/r.js -o src/main/js/requirejs-ace-config.js logLevel=2 out=build/opt-resources/xtext-ace.min.js optimize=uglify preserveLicenseComments=false",
+		"optimizeAceCss": "node node_modules/requirejs/bin/r.js -o logLevel=2 cssIn=src/main/css/xtext/xtext-ace.css out=build/opt-resources/xtext-ace.css",
 		"optimizeOrion": "npm run optimizeOrionJs && npm run optimizeOrionJsMin && npm run optimizeOrionCss",
-		"optimizeOrionJs":"r.js -o src/main/js/requirejs-orion-config.js logLevel=2 out=build/opt-resources/xtext-orion.js optimize=none",
-		"optimizeOrionJsMin": "r.js -o src/main/js/requirejs-orion-config.js logLevel=2 out=build/opt-resources/xtext-orion.min.js optimize=uglify preserveLicenseComments=false",
-		"optimizeOrionCss": "r.js -o logLevel=2 cssIn=src/main/css/xtext/xtext-orion.css out=build/opt-resources/xtext-orion.css",
+		"optimizeOrionJs":"node node_modules/requirejs/bin/r.js -o src/main/js/requirejs-orion-config.js logLevel=2 out=build/opt-resources/xtext-orion.js optimize=none",
+		"optimizeOrionJsMin": "node node_modules/requirejs/bin/r.js -o src/main/js/requirejs-orion-config.js logLevel=2 out=build/opt-resources/xtext-orion.min.js optimize=uglify preserveLicenseComments=false",
+		"optimizeOrionCss": "node node_modules/requirejs/bin/r.js -o logLevel=2 cssIn=src/main/css/xtext/xtext-orion.css out=build/opt-resources/xtext-orion.css",
 		"optimizeCodeMirror": "npm run optimizeCodeMirrorJs && npm run optimizeCodeMirrorJsMin && npm run optimizeCodeMirrorCss",
-		"optimizeCodeMirrorJs":"r.js -o src/main/js/requirejs-codemirror-config.js logLevel=2 out=build/opt-resources/xtext-codemirror.js optimize=none",
-		"optimizeCodeMirrorJsMin": "r.js -o src/main/js/requirejs-codemirror-config.js logLevel=2 out=build/opt-resources/xtext-codemirror.min.js optimize=uglify preserveLicenseComments=false",
-		"optimizeCodeMirrorCss": "r.js -o logLevel=2 cssIn=src/main/css/xtext/xtext-codemirror.css out=build/opt-resources/xtext-codemirror.css"
+		"optimizeCodeMirrorJs":"node node_modules/requirejs/bin/r.js -o src/main/js/requirejs-codemirror-config.js logLevel=2 out=build/opt-resources/xtext-codemirror.js optimize=none",
+		"optimizeCodeMirrorJsMin": "node node_modules/requirejs/bin/r.js -o src/main/js/requirejs-codemirror-config.js logLevel=2 out=build/opt-resources/xtext-codemirror.min.js optimize=uglify preserveLicenseComments=false",
+		"optimizeCodeMirrorCss": "node node_modules/requirejs/bin/r.js -o logLevel=2 cssIn=src/main/css/xtext/xtext-codemirror.css out=build/opt-resources/xtext-codemirror.css"
 	}
 }

